app/scripts/migrations/137.test.ts

```diff
@@ -516,8 +497,194 @@ describe('migration #137', () => {
           domains: {},
         },
       });
+      const basePermission = {
+        id: '1',
+        date: 2,
+        invoker: 'test.com',
+        parentCapability: PermissionNames.eth_accounts,
```

Comment:
mmm, should this property exist here? or should the var be renamed? nit

Reply:
yeah... I was just putting it there since its the most common one and then spread and overwritten for other cases

Reply:
Happy to change if it bugs ya tho

Reply:
changed it to `baseEthAccountsPermissionMetadata`  ?
https://github.com/MetaMask/metamask-extension/pull/29682/commits/e1b700671e105bc15064ee35eea21db41ba10bfd